webapp/config/httpd.conf

```diff
@@ -168,6 +168,9 @@ DocumentRoot "{{pkg.svc_data_path}}/htdocs"
     # Controls who can get stuff from this server.
     #
     Require all granted
+
+    RewriteEngine  on
+    RewriteRule    "^(.*)"  "/cgi-bin/hello-world"
```

Comment:
This makes it so you can request anything and it will automatically redirect to the Habby page.